### DISCUSSION TITLE
core 2024.4.0

### DISCUSSION THREAD

**Discussion by miami71it** [indomus/forum#3021] [GENERALE - Discussioni generiche]
ciao a tutti dopo l'aggiornamento in oggetto , tutte le icone fatte con buttom car custom sono diventare quadrate e nn piu' con spigoli tondi, come mai? cosa posso fare per farle tornare come prima?

**Comment by snakuzzo** [comment]
Se usi un custom, guarda tra le issue del custom su github

**Reply by miami71it** [reply]
grazie, ma sai per caso dove? io non uno mai github

**Reply by snakuzzo** [reply]
io come cavolo faccio a saperlo ?
Saprai tu quale custom usi. Cercalo su google e troverai il link al progetto github. Vai lì e cerchi tra le issue.
Cercando con quello che hai scritto...viene fuori [QUESTO](https://github.com/custom-cards/button-card)

**Reply by miami71it** [reply]
si e' quello , io pensavo avevi il link del forum corretto pardon non ci siamo capiti
pero' questo e' il custom ma il mio problema rimane, in quanto sono poco esperto non riesco a capire cosa dovrei modificare dopo l'aggiornamento in quanto i tasti sono diventati guardarati e non piu' con gli spigoli rotondi. Ci studiero' su vediamo se ci capisco.

**Reply by snakuzzo** [reply]
sempre lì, vedi bene che nella pagina trovi il link alle issue. Vai lì e cerchi. Più di questo non so dirti
